js/core.js

```diff
@@ -80,6 +80,8 @@ session.on("torrent.add", function(e) {
         };
 
         metadata["options"] = options;
+
+        logger.debug("Torrent '' assigned Seed Ratio: " + options.seedRatio + "%, Seed Time: " + options.seedTime + "minutes , Seed Action: " + options.seedAction);
```

Comment:
It looks like the name is not available when this callback is made. I think the best I can make it print is the infohash, similar to what some of the other debug lines spew.